core/backend/src/IModelExporter.ts

```diff
@@ -750,16 +750,16 @@ class ChangedInstanceIds {
   public relationship = new ChangedInstanceOps();
   public font = new ChangedInstanceOps();
   private constructor() { }
-  public static async initialize(requestContext: AuthorizedClientRequestContext, iModelDb: BriefcaseDb, startChangeSetId: string): Promise<ChangedInstanceIds> {
+
+  public static async initialize(requestContext: AuthorizedClientRequestContext, iModel: BriefcaseDb, firstChangeSetId: string): Promise<ChangedInstanceIds> {
     requestContext.enter();
-    const extractContext = new ChangeSummaryExtractContext(iModelDb); // NOTE: ChangeSummaryExtractContext is nothing more than a wrapper around IModelDb that has a method to get the iModelId
-    // NOTE: ChangeSummaryManager.downloadChangesets has nothing really to do with change summaries but has the desired behavior of including the start changeSet (unlike BriefcaseManager.downloadChangesets)
-    const changeSets = await ChangeSummaryManager.downloadChangesets(requestContext, extractContext, startChangeSetId, iModelDb.changeSetId);
+
+    const changeSets = await ChangeSummaryManager.downloadChangesets(requestContext, iModel.iModelId, firstChangeSetId, iModel.changeSetId);

```

Comment:
this should be replaced by a call to `hubAccess.downloadChangesets` using ChangesetRange